src/HyperFormula.ts

```diff
@@ -2699,26 +2754,30 @@ export class HyperFormula implements TypedEmitter {
    *
    * @category Helpers
    */
-  public getCellDependents(address: SimpleCellAddress | AbsoluteCellRange): (AbsoluteCellRange | SimpleCellAddress)[] {
+  public getCellDependents(address: SimpleCellAddress | SimpleCellRange): (SimpleCellRange | SimpleCellAddress)[] {
```

Comment:
@kirszenbaum Probably for another issue, should we add another example with SimpleCellRange as input? Currently only SimpleCellAddress is demonstrated

Reply:
@wojciechczerniak - sure, I added a note to the backlog: https://github.com/handsontable/hyperformula/projects/1#card-63587403